example_tournament.py

```diff
@@ -6,14 +6,17 @@
 
 def run_tournament():
     tournament = tilewe.tournament.Tournament([
-        tilewe.engine.MaximizeMoveDifferenceEngine(), 
-        tilewe.engine.LargestPieceEngine(), 
+        tilewe.engine.MoveDifferenceEngine("MaxMoveDiff", "max"), 
+        tilewe.engine.MoveDifferenceEngine("MinMoveDiff", "min"), 
+        tilewe.engine.PieceSizeEngine("LargestPiece", "max"), 
+        tilewe.engine.PieceSizeEngine("SmallestPiece", "min"), 
         tilewe.engine.TileWeightEngine("WallCrawler", 'wall_crawl'),
         tilewe.engine.TileWeightEngine("Turtle", 'turtle'),
-        tilewe.engine.MostOpenCornersEngine(), 
+        tilewe.engine.OpenCornersEngine("MostOpenCorners", "max"), 
```

Comment:
All these ones with "max" don't actually need args since it's the default, but included for clarity